docs/features/software-catalog/external-integrations.md

```diff
@@ -318,6 +332,98 @@ backend.add(catalogModuleFrobsProvider);
 backend.start();
 ```
 
+#### Follow-up: Config Defined Schedule
```

Comment:
Should we also mention using the `config.d.ts` file here? Could be as simple as pointing to the docs on the topic